perfkitbenchmarker/linux_packages/redis_server.py

```diff
@@ -15,11 +15,25 @@
 
 """Module containing redis installation and cleanup functions."""
 
+from perfkitbenchmarker import flags
 from perfkitbenchmarker import vm_util
 
-REDIS_TAR = 'redis-2.8.9.tar.gz'
-REDIS_DIR = '%s/redis-2.8.9' % vm_util.VM_TMP_DIR
+
+flags.DEFINE_integer('redis_total_num_processes', 1,
+                     'Total number of redis server processes.')
+
+flags.RegisterValidator(
```

Comment:
nit: you can just use lower_bound in the flag definition instead of registering a validator if this is the only check you are performing (see https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/linux_benchmarks/netperf_benchmark.py#L43 for an example)